frontend/src/components/SignUp.jsx

```
@@ -75,94 +83,108 @@ const SignUp = () => {
+            <div className="w-full h-[1px] bg-gray-300"></div>
+            <p>or</p>
+            <div className="w-full h-[1px] bg-gray-300"></div>
           </div>
-          <p className='text-sm text-center mt-4'>
+          <GoogleButton
+            variant="outlined"
+            size="large"
+            onClick={handleGoogleLogin}
+            disabled={googleLoading}
+            startIcon={
+              <svg width="18" height="18" viewBox="0 0 24 24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)